Extensions/xcloud.js

```diff
@@ -12,6 +12,15 @@ XKit.extensions.xcloud = new Object({
 	username: "",
 	password: "",
 
+
+	get_xcloud_url: function() {
+		if (XKit.extensions.xcloud.useoldserver === true) {
```

Comment:
`if (XKit.extensions.xcloud.useoldserver)`

There's other places in this code that can use this convention as well.